xray/.gitignore

```diff
@@ -0,0 +1 @@
+results/
```

Comment:
I've been adding other xray-related ignores to the top level `.gitignore` so far. I like this idea of having them in the xray folder, but it would make it inconsistent. I'll leave it to you to decide if you move this to the top-level or if you move the other xray-related ones here, but I would like to see it fixed

Reply:
Good point! 💯

Reply:
Moved to top level `.gitignore`